src/sentry/static/sentry/app/components/group/times.jsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import styled from 'react-emotion';
+import PropTypes from 'prop-types';
+
+import {t} from 'app/locale';
+import InlineSvg from 'app/components/inlineSvg';
+import TimeSince from 'app/components/timeSince';
+import space from 'app/styles/space';
+import overflowEllipsis from 'app/styles/overflowEllipsis';
+
+/**
+ * Renders the first & last seen times for a group or event with
+ * a clock icon.
+ */
+const Times = props => {
+  const {lastSeen, firstSeen} = props;
+  return (
+    <Container>
+      <div css={overflowEllipsis}>
+        {lastSeen && (
+          <React.Fragment>
+            <GroupTimeIcon src="icon-clock-sm" />
+            <TimeSince date={lastSeen} suffix={t('ago')} />
+          </React.Fragment>
+        )}
+        {firstSeen && lastSeen && (
+          <span className="hidden-xs hidden-sm">&nbsp;—&nbsp;</span>
+        )}
+        {firstSeen && (
+          <TimeSince date={firstSeen} suffix={t('old')} className="hidden-xs hidden-sm" />
+        )}
+      </div>
+    </Container>
+  );
+};
+Times.propTypes = {
+  lastSeen: PropTypes.string,
```

Comment:
Are these ever going to be Date objects?

Reply:
I haven't seen that scenario yet.